README.md

```diff
@@ -1,6 +1,14 @@
-[![Build Status](https://travis-ci.org/aws/aws-xray-sdk-python.svg?branch=master)](https://travis-ci.org/aws/aws-xray-sdk-python)
+![Build Status](https://github.com/aws/aws-xray-sdk-python/actions/workflows/IntegrationTesting.yaml/badge.svg)
 [![codecov](https://codecov.io/gh/aws/aws-xray-sdk-python/branch/master/graph/badge.svg)](https://codecov.io/gh/aws/aws-xray-sdk-python)
 
+### :mega: OpenTelemetry Python with AWS X-Ray
+
+AWS X-Ray recommends using AWS Distro for OpenTelemetry (ADOT) to instrument your application **instead of this X-Ray SDK** due to its wider range of features and instrumentations. See the [AWS X-Ray docs on Working with Python](https://docs.aws.amazon.com/xray/latest/devguide/xray-python.html) for more help with choosing between ADOT and X-Ray SDK.
+
+**NOTE**: OpenTelemetry Python **does not currently support Centralized Sampling**, only the X-Ray SDK does, but we hope to have this feature in OTel the future.
```

Comment:
"...have this feature in OTel the future"
Should be "have this feature in OTel in the future"